source/instructors/setupcourses/assignment-settings/allow-regrade-request.rst

```diff
@@ -7,29 +7,30 @@
+You can allow students to request for a regrade of their assignments. This gives students an opportunity to voice concerns about their grades while enabling teachers to address those concerns transparently. It also fosters a more collaborative learning environment.
 
-To allow regrade request by student, follow these steps:
+To allow students to request regrades, follow these steps:
 
 1. Open the assignment **Settings**.
-2. In the **Assignment** settings, click the **Allow Regrade Request by Student**.
+2. In the **Assignment** settings, enable **Allow Regrade Request by Student**.
 3. Click **Save Changes**.
 
-Regrade request option will be available to students only after their assignments are completed/graded and grades are released.
++The regrade request option will be available to students only after their assignments are marked complete and graded, and grades are released.
 
-Once a student requested for regrade, by default all teachers in the course will receive a notification email regarding this. If you don't want to send emails to all the teachers then you can turn off the notification emails or customize who should receive these emails from **Grading/Basic Settings** tab. These requests will also appear in the grading queue and on the assignment progress page as well
+Once a student requests a regrade, all teachers in the course will receive a notification email by default. If you don't want all teachers to receive these emails, you can turn off notifications or customize recipients in the **Grading/Basic Settings** tab. Regrade requests will also appear in the grading queue and on the assignment progress page.
 
-
-  .. image:: /img/regrade-request-on-progress.png
-     :alt: Regrade request showing on Prograss page
+.. image:: /img/regrade-request-on-progress.png
+   :alt: Regrade request showing on Progress page
 
 Click on it to open the grading dialog and see the regrade request with the student's message from the **Regrade Request** section.
 
-  .. image:: /img/regrade-request-grade-dialog.png
-     :alt: Grade Dialog
+.. image:: /img/regrade-request-grade-dialog.png
+   :alt: Grade Dialog
 
-From this grading dialog you can: 
+From this grading dialog, you can: 
 
-- Open the student's project involved using the **Open Project** button
+- Open the student's project using the **Open Project** button
 - **Override Grade** of the student (once you override the grade, the regrade request will be automatically removed)
-- **Reject** the regrade request with a message for student 
\ No newline at end of file
+- **Reject** the regrade request with a message for the student
+
+.. note:: If you have enabled resubmission after marking as complete, students will automatically be prompted to request a regrade when they resubmit their work.
```

Comment:
I think the wording here might be more correct as ...students will have the option to request a regrade when they resubmit their work